src/TextField/TextField.js

```diff
@@ -115,6 +126,11 @@ export default class TextField extends Component {
           disabled={disabled}
           {...inputProps}
         />
+        {helperText && (
+          <FormHelperText className={helperTextClassName}>
```

Comment:
maybe pass error to FormHelperText make it support error style like 
<FormHelperText>{errorText || helperText}</FormHelperText>
what do you think @oliviertassinari @kybarg